Problem2.java

```diff
@@ -0,0 +1,46 @@
+//PROBLEM - 205. Isomorphic Strings
+/** 3 Pointer Approcach: 
+ * Use 2 <Key,Value> of <CHAR,CHAR> hash sets ad HashMap data structure best suitable to hold K,V data.
+ * APPROACH: 
+ * 1. If Character1/Character2 is present as key then check corresponding Character with current Character.
+ * 2. If match keep moving, if not retuen FALSE.
+ * 3. Incase c/c not present add to HashMap.
+ */ 
+//
+//time Complexity :
+// O(N)
```

Comment:
O(n^2)
Looks like you copied the template but forgot to edit it for this problem!